docs.css

```diff
@@ -0,0 +1,87 @@
+.logo--header {
+  width: 4rem;
+}
+
+header {
+  padding: 1rem;
+  display: flex;
+  column-gap: 20px;
+  align-items: center;
+  grid-area: header;
+  box-shadow: var(--green-teal) 2px 5px 5px;
+}
+
+body {
+  background-color: var(--background-color);
+  display: grid;
+  width: 100vw;
+  height: 100vh;
+  grid-template-areas: "header header" "aside main";
+  grid-template-rows: 5rem 1fr;
+  grid-template-columns: 15%;
+  box-sizing: border-box;
+}
+
+main {
```

Comment:
Avoid grabbing elements in a component library, try using classes in place of it. Doing so can affect the user's styling.